hasura/metadata/databases/default/tables/public_api_key.yaml

```diff
@@ -21,6 +21,15 @@ insert_permissions:
         - name
         - team_id
 select_permissions:
+  - role: api_key
+    permission:
```

Comment:
Do we still need these permissions? We are using service client to check this right?

Reply:
Removed